documentation/1.0/content/setup/install.md

```diff
@@ -9,6 +9,60 @@ description: "Install the WKT UI application and check for updates."
 1. Download the latest WebLogic Kubernetes Toolkit UI (WKT UI) application installers from the [GitHub Releases section](https://github.com/oracle/weblogic-toolkit-ui/releases) of this repository.
 2. Simply run the appropriate installer for your operating system.
 
+**NOTE**: When installing on Linux, to get _all_ the dependencies and installed in the correct order, use a command similar to the following:
+```
+sudo yum localinstall wktui-1.0.0.x86_64.rpm
+```
+
+- On Oracle/RedHat/CentOS (and some others), use either `yum` or `dnf`
```

Comment:
I would move this up above the `sudo yum ...` line since yum doesn't exist on Ubuntu...

Reply:
ok, got it.